benchmarks/smallmatvec/manual_versions/libxsmm/runme.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Note, this script has Intel-specific KMP environment variables in it
+
+# This script executes the selected original and xsmm matrix_vector
+# multiplication implementation with a variety of OpenMP
+# configurations to analyse its performance.
+
+# Update the for statements to configure each parameter search space.
+
+# The configurable parameters are:
+#   - $ps: problem size (horizontal)
+#   - $vertical: problem size (vertical)
+#   - $omp_sch: OpenMP scheduler [static, dynamic, guided]
+#   - $omp_aff: OpenMP affinity [none, compact, scatter]
+#   - $smt: Number of SMT
+#   - $cores: Number of cores
+#   - $v: matrix_vector implementation [orig, xsmm, reorder]
+
+# echo "Don't forget to set the parameters to the required values"
+# exit
+
+echo "Original vs libxsmm timing results"
+
+for ps in 32; do
```

Comment:
Does libxsmm use OpenMP? The only OpenMP I saw in the source was for timing and for SIMD?

Reply:
All tests use the same driver (kdriver.f90) which has OpenMP over cells with colouring. Does that answer the question?